cmd/node/node.go

```diff
@@ -339,11 +340,14 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 	idStore := activation.NewIdentityStore(iddbstore)
 	poetDb := activation.NewPoetDb(poetDbStore, lg.WithName("poetDb"))
 	//todo: this is initialized twice, need to refactor
+
 	validator := nipst.NewValidator(commitmentConfig, poetDb)
 	mdb := mesh.NewPersistentMeshDB(dbStorepath, lg.WithName("meshDb"))
 	atxdb := activation.NewActivationDb(atxdbstore, nipstStore, idStore, mdb, app.Config.CONSENSUS.LayersPerEpoch, validator, lg.WithName("atxDb"))
+
 	beaconProvider := &oracle.EpochBeaconProvider{}
-	blockValidator := oracle.NewBlockEligibilityValidator(int32(layerSize), uint16(layersPerEpoch), atxdb, beaconProvider, BLS381.Verify2, lg.WithName("blkElgValidator"))
+	eligibilityValidator := oracle.NewBlockEligibilityValidator(int32(layerSize), uint16(layersPerEpoch), atxdb, beaconProvider, BLS381.Verify2, lg.WithName("blkElgValidator"))
+	blockValidator := sync.NewBlockValidator(eligibilityValidator, sync.TxValidatorMock{}, sync.AtxValidatorMock{})
```

Comment:
Why all of them are mocks?

Reply:
waiting for implementation from gavra and noam

Reply:
ATX validation already exists: see `SyntacticallyValidateAtx` in `atxdb.go`
I believe the only validation on a tx is it's sig which @zalmen has implemented